mage_integrations/requirements.txt

```diff
@@ -8,6 +8,7 @@ google-analytics-data==0.14.2
 google-api-python-client==2.70.0
 google-cloud-bigquery-storage==2.16.2
 google-cloud-bigquery~=3.0
+google-cloud-storage==2.8.0 
```

Comment:
Let's use version 2.5.0 here to make it consistent with this file: https://github.com/mage-ai/mage-ai/blob/master/requirements.txt#L51